ComponentKit/Core/Scope/CKComponentScopeHandle.mm

```diff
@@ -38,9 +38,7 @@ + (CKComponentScopeHandle *)handleForComponent:(CKComponent *)component
 
   CKComponentScopeHandle *handle = currentScope->stack.top().frame.handle;
   if ([handle acquireFromComponent:component]) {
-    if (CKSubclassOverridesSelector([CKComponent class], [component class], @selector(boundsAnimationFromPreviousComponent:))) {
-      [currentScope->newScopeRoot registerBoundsAnimationComponent:component];
-    }
+    [currentScope->newScopeRoot registerComponent:component];
```

Comment:
Cool, so register components with the scope root regardless of what it's actually being used for.